ClimsoftVer4/ClimsoftVer4/ucrTextBox.vb

```diff
@@ -276,38 +296,55 @@ Public Class ucrTextBox
         OnevtTextChanged(sender, e)
 
     End Sub
-
+    ''' <summary>
+    ''' Sets the focus to the control 
+    ''' </summary>
     Public Sub GetFocus()
         txtBox.Focus()
     End Sub
-
+    ''' <summary>
+    ''' Checks if a textbox is empty
+    ''' Returns true when text box is empty
+    ''' </summary>
+    ''' <returns></returns>
     Public Function IsEmpty() As Boolean
         If TextboxValue.Length > 0 Then
             Return False
         Else
             Return True
         End If
     End Function
```

Comment:
Since you've edited this control though not necessary but it would be great to have this changed;

from 
```
If TextboxValue.Length > 0 Then
             Return False
         Else
             Return True
         End If
```

To just

`Return (TextboxValue.Length > 0)`

I didn't see this redundancy when I was writingthe code.